worker/src/rebuild.rs

```diff
@@ -128,16 +139,43 @@ pub async fn rebuild(ctx: &Context<'_>, url: &str) -> Result<Rebuild> {
 
     // process result
     let output_path = out_dir.join(&filename);
+
```

Comment:
In this file, I think this is where the first change should be (apart from the imports).